common/package.json

```diff
@@ -1,3 +1,7 @@
 {
-    "name": "tetrix-common"
+    "name": "tetrix-common",
+    "scripts": {
+        "prepare": "tsc -b",
```

Comment:
`prepare` is run for every library that is installed as a dependency when the downstream code is running `npm install`. It will build the code to ./build folder, and that folder will be the code that server actually uses.